src/Component/spec/Config/Builder/FieldSpec.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
```

Comment:
Minor thing, but we usually add it 
```suggestion

declare(strict_types=1);

```